tests/Helpers.php

```diff
@@ -14,6 +14,17 @@
 // Mock escaping function.
 Functions\when('wp_kses_post')->returnArg();
 
+// Mock json success and error handlers.
+Functions\when('wp_send_json_success')->alias(function(...$args) {
+    echo json_encode($args);
+});
```

Comment:
```suggestion
Functions\when('wp_send_json_success')->alias(function($data = null, $status_code = null, $options = 0) {
    $response = array( 'success' => true );
    
    if ( isset( $data ) ) {
        $response['data'] = $data;
    }
    
    echo json_encode($response, $options);
});
```

`wp_send_json_success` should add `success => true` to the response. Also I know this is simplified mock but it doesn't hurt to be true to the source, what do you think? https://developer.wordpress.org/reference/functions/wp_send_json_success/

Reply:
Sure thing, I'm fine with it. I opted for a simplified version because I'm not really interested in what is going on under the hood of the `wp_send_json_success` function (at the end it's just `echo` the `json_encode` and `die` with some custom headers set). The unit test is only concerned that the method we implemented works as it should.

But I guess this mock cannot hurt 👍🏼  I'll update it and the tests accordingly.

Reply:
Yeah I 100% understand. Only mentioned this because I could imagine a scenario where I write a test for an arbitrary route and assert that `success` key is set to ensure it was called successfully (as you'd test this in your app code).